generated/nifake/session.py

```diff
@@ -349,7 +349,7 @@ def read_from_channel(self, maximum_time):
             session.channels['0,1'].read_from_channel(maximum_time)
 
         Args:
-            maximum_time (datetime.timedelta): Specifies the **maximum_time** allowed in microseconds.
+            maximum_time (datetime.timedelta): Specifies the **maximum\_time** allowed in microseconds.
```

Comment:
I didn't expect backslashes in the Python code, seems like several got added.

Reply:
Not added. No longer removed. Updated hand maintained documentation.